fe/fe-core/src/test/java/org/apache/doris/analysis/ExprTest.java

```diff
@@ -24,18 +24,43 @@
 
 import com.google.common.collect.Maps;
 
+import org.apache.doris.mysql.privilege.MockedAuth;
+import org.apache.doris.mysql.privilege.PaloAuth;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.utframe.DorisAssert;
+import org.apache.doris.utframe.UtFrameUtils;
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.BeforeClass;
+import org.junit.AfterClass;
 
-import java.util.Map;
-import java.util.Set;
+import java.io.IOException;
+import java.util.*;
 
 import mockit.Expectations;
 import mockit.Injectable;
 import mockit.Mocked;
 
 public class ExprTest {
 
+    private static String runningDir = "fe/mocked/DemoTest/" + UUID.randomUUID().toString() + "/";
```

Comment:
You could add the test in `SelectStmtTest`, and then you could directly write `testRandFunction `